app/models/inspection.rb

```diff
@@ -1,6 +1,7 @@
 class Inspection < ActiveRecord::Base
-  attr_accessible :inspection_date, :person_id, :status
-  
-  belongs_to :person
+
+  belongs_to :item
+
+  STATUS_CHOICES = ['Complete - Passed', 'Complete - Failed', 'Incomplete']
```

Comment:
In looking at this, I realize this list is incomplete, as you could have "Incomplete - Failed". (If the engine is seized, don't bother checking the turn signals on a vehicle) 
Seeing the new list, it suggests that there are really two concepts here, completion, and results. You just can't have "Incomplete - Passed". Something for us to consider for a later enhancement.